packages/bindings/Cargo.toml

```diff
@@ -14,7 +14,9 @@ homepage = "https://www.desmos.network"
 keywords = ["social-networks", "blockchain", "smart-contracts", "desmos", "cryptocurrency"]
 
 [dependencies]
+anyhow = "1.0.58"
 cosmwasm-std = { version = "1.0.0" }
+cw-multi-test = { version = "0.13.4", optional = true }
```

Comment:
Not sure why it has conflict with `test-contract` cosmwasm-std, so I make it optional and only use it when the feature is `mock`.

Reply:
I tried to remove the `optional` flag and the dependency requirement with the `mocks` feature and on my computer running `cargo build` everiting compiles correctly. 
Can you tell me the command that triggers the error for you?

I'm using:
* cargo 1.61.0 (a028ae4 2022-04-29)
* rustc 1.61.0 (fe5b13d68 2022-05-18)

Reply:
The error happens while compileing the test contract with command `cargo optimize`.
It shows error as follows:
```
   Compiling cw-multi-test v0.13.4
error[E0432]: unresolved import `cosmwasm_std::testing`
 --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/cw-multi-test-0.13.4/src/app.rs:5:19
  |
5 | use cosmwasm_std::testing::{mock_env, MockApi, MockStorage};
  |                   ^^^^^^^ could not find `testing` in `cosmwasm_std`

error[E0432]: unresolved import `cosmwasm_std::testing`
  --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/cw-multi-test-0.13.4/src/wasm.rs:24:19
   |
24 | use cosmwasm_std::testing::mock_wasmd_attr;
   |                   ^^^^^^^ could not find `testing` in `cosmwasm_std`
```

Both cargo and rustup version on my PC are 1.64.0-nightly.

Reply:
Ok found the issue, becouse  `cw-multi-test` is marked as dependency inisde the `desmos-binding` toml it's compiled also when targeting `wasm32-unknown-unknown`.
`cosmwasm-std` expose the `testing` module only when not compiling for `wasm32-unknown-unknown` causing this compilation error.
Moving the `cw-multi-test` into the `dev-dependencies` will fix this issue